### ISSUE TITLE
Please provide a slider / parameter for transparency

### ISSUE THREAD

**Issue by Sithis696** [extesy/DeckTracker#27]
Hello Extesy,

could you enable an option to let us set the alpha channel for the overlay? I would like to see the animations not being cut off and a continuous board (and just am a big fan of transparency). =)

Thank you so much for the great work, I love it, keep it up!

**Comment by extesy** [comment]
Yes, this will be part of the configuration UI which is missing at the moment.

**Comment by extesy** [comment]
@Sithis696 Second request has been implemented in `1.0.54`.

**Comment by Sithis696** [comment]
Just saw it - amazing work! Thank you so much!

**Comment by andr9528** [comment]
Would be nice with an option/ability to move the deck tracker to a screen that doesn't necessarily run the game - I have 2 screens

**Comment by extesy** [comment]
@andr9528 Unfortunately that is not possible unless the game itself supports two monitors.
